docs/configuring.md

```diff
@@ -365,6 +365,14 @@ Don't forget to read [the docs about the answers file](#the-copier-answersyml-fi
     _answers_file: .my-custom-answers.yml
     ```
 
+### `config_file`
+
+-   Format: `str`
+-   CLI flags: `-c`, `--config-file`
+-   Default value: `copier.y?ml`
```

Comment:
OK might not be a typo, but the `?` wildcard is usually put *after* the character it affects, not instead of it. Just a nit pick,

Reply:
You are right, I have changed it to explicit two values